packages/iov-ethereum/src/serialization.ts

```diff
@@ -64,67 +68,43 @@ export class Serialization {
   public static serializeUnsignedTransaction(
     unsigned: UnsignedTransaction,
     nonce: Nonce,
-    erc20Tokens: ReadonlyMap<TokenTicker, Erc20Options> = new Map(),
-    atomicSwapEtherContractAddress?: Address,
-    atomicSwapErc20ContractAddress?: Address,
+    erc20Tokens: Erc20TokensMap = new Map(),
+    atomicSwapContractAddress?: Address,
   ): Uint8Array {
     const chainIdHex = Serialization.getChainIdHex(unsigned);
     const gasPriceHex = Serialization.getGasPriceHex(unsigned);
     const gasLimitHex = Serialization.getGasLimitHex(unsigned);
 
+    const args: [string, string, string, Nonce, Erc20TokensMap, Address?] = [
+      chainIdHex,
+      gasPriceHex,
+      gasLimitHex,
+      nonce,
+      erc20Tokens,
+      atomicSwapContractAddress,
+    ];
```

Comment:
If `serializeUnsignedSendTransaction`, `serializeUnsignedSwapOfferTransaction`, `serializeUnsignedSwapClaimTransaction`, `serializeUnsignedSwapAbortTransaction`, `serializeUnsignedErc20ApproveTransaction` should share the same interface (which makes sense, looking at this code), I think we should pass an onject as the second paramter instead of creating dummy positional arguments.

Reply:
I thought about this but wasn't sure if we have a policy of when to use arguments vs objects?

Reply:
There is no strict policy, but I'd say we use multiple positional arguments by default unless we have a very good reason. In this case we have a very good reason: 5 different function calls should be fed with arguments from a single source.